csharp-basic-topics/RemoveIllegalCharsFromPathAndFilenames/RemoveIllegalCharsFromPathAndFilenames/Program.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System.Text.RegularExpressions;
+
+namespace CodeMazeGuides.CSBasic
+{
+    public class Program
+    {
+        static void Main(string[] args)
+        {
+            List<string> pathList = new List<string> 
+            {
+                "C://Thisisavalidpath",
+                "D://Thi*IsINVALID/<<",
+                "E://Users/ValidPath/docs",
+                "F://Users/Photos/Cats",
+                "G://Inv@7***lidPath",
+                "H://Test/This<IS>Ano<>therInvalidPath"
+            };
+            var convCheckResult = GetInvalidPaths(pathList);
```

Comment:
All of these are providing different results, right? We should output all of them and not only the LINQ check result.